app/assets/javascripts/users_select.js.coffee

```diff
@@ -107,7 +107,6 @@ class @UsersSelect
         per_page: 20
         active: true
         project_id: @projectId
-        group_id: @groupId
```

Comment:
We don't want to disable this entirely, since it _is_ needed in some places. We just need `scope: :all` to be added to the `users_select_tag` call in `app/views/admin/groups/show.html.haml`.